contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,257 @@
+
+### Why should you care about churn?
+
+Reducing churn makes intuitive sense, but it's worth considering the practical impact of high churn and reducing it. You can boil it down to three points:
+
+  1. High churn is a likely sign you don't have product-market fit.
+
+  2. High churn leads to increased sales and/or marketing spend to compensate.
+
+  3. Reducing churn compounds – i.e. revenue **and** marketing ROI increase.
+
+All are compelling reasons to reduce churn, but you need some context before you do. Is your current churn rate actually bad, or does it just **feel** bad? Let's dig into that next.
+
+### What is a good churn rate?
+
+For research, I read several dozen data analyses and finger-in-the-air takes on what "good churn" looks like. Suffice it to say there are contrasting views on the topic.
+
+I've collated the better sources [in the appendix](#more-on-churn-rate-benchmarks). What follows is my own, not-quite-scientific meta-analysis:
+
+- **B2C SaaS:** Subscription services like Netflix or Tidal typically have higher churn than those targeting businesses, so a 3 to 5% monthly churn rate is pretty good. Anything less than 3% is very good, though B2C services are more price sensitive – churn trends much lower among services that charge less than $10 per month, for example. A monthly churn rate of 5% is equivalent to annual churn of 46% – i.e. a company with 1,000 customers would lose 460 over 12 months.
+
+- **Mid-Market B2B SaaS:** Anything up to 5% monthly churn is OK for a B2B SaaS that **doesn't** sell to big enterprises, but something around 3% is preferable if your average revenue per customer (ARPC) is above $250 per month. Anything below 2% is excellent. A monthly churn rate of 3% is equivalent to 30.6% annually – i.e. a company with 1,000 customers would lose 306 over 12 months.
+
+- **Early-stage B2B SaaS:** A monthly churn rate above 5% is to be expected for any early-stage company B2B SaaS, so don't worry too much if you're growing fast and you're above that benchmark. Just monitor your trend and dig deeper if you don't see it decline as you mature. A monthly churn of 5.65% is equivalent to 50.24% annually – i.e. a company with 1,000 companies would lose 502 over 12 months, more than half.
+
+- **Enterprise B2B SaaS:** Products selling to large enterprises (think more than 1,000 employees) have lower churn, mainly because large companies change slowly and will generally bill annually. A monthly churn rate up to 2% is good. Anything less than 1% is excellent. A monthly churn rate of 1% is equivalent to 11.3% annually – i.e. a company with 1,000 customers will lose 114 over 12 months.
```

Comment:
```suggestion
- **Enterprise B2B SaaS:** Products selling to large enterprises (think more than 1,000 employees) have lower churn, mainly because large companies change slowly and are billed annually. A monthly churn rate up to 2% is good. Anything less than 1% is excellent. A monthly churn rate of 1% is equivalent to 11.3% annually – i.e. a company with 1,000 customers will lose 114 over 12 months.
```